pkgs/by-name/ja/jaero/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  libsForQt5,
+  libacars,
+  libcorrect,
+  libvorbis,
+  zeromq,
+  qmqtt,
+  makeWrapper,
+}:
+
+let
+  jfft-src = fetchFromGitHub {
+    owner = "jontio";
+    repo = "JFFT";
+    rev = "4b74486e58e1d266f1cc3c570f3d073d40c353d6";
+    hash = "sha256-GDagnbbYT6Xn8IQ7YH7giBiNqSYYkyyTqInlvT5TZZA=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jaero";
+  version = "v1.0.4.14-unstable-2023-11-21";
```

```suggestion
  version = "1.0.4.14-unstable-2023-11-21";
```
may i ask why the latest commit and not the latest version?